master/package.py

```diff
@@ -3,6 +3,7 @@
     'CPPFLAGS': None,
     'LLVM_CMAKE': util.Property('llvm_cmake', default=None),
     'MACOS_CODESIGN_IDENTITY': MACOS_CODESIGN_IDENTITY,
+    'INNO_ARGS': '/Dsign=true "/Smysigntool=\$$q`cygpath -w /bin/sh`\$$q -c ~/sign.sh \$$f"',
```

Comment:
Something looks off here.

```
`cygpath -w /bin/sh`
```
should probably just be

`bin/sh`

Reply:
Not sure what this does or if it will work:
`-c ~/sign.sh`

Reply:
`cygpath -w /bin/sh` will turn it into something like `C:\cygwin\bin\sh`, which is necessary because Inno is not a cygwin application, so any cygwin path needs to be turned into a windows path.

I believe the `\$$q` thing is a quote, which I'm using here just because I don't want to assume that cygwin is installed in a directory that does not have a space in its path.  The end result should be something like:

```
/Dsign=true "/Smysigntool=\$$qC:\cygwin\bin\sh\$$q -c ~/sign.sh \$$f"
```

Which then results in `mysigntool` having the value:
```
"C:\cygwin\bin\sh" -c ~/sign.sh $f
```

Which I think is what we want.

Reply:
In that case I think you need to move `$$q` to the end
```
/Dsign=true "/Smysigntool=\$$qC:\cygwin\bin\sh\ -c ~/sign.sh$$q \$$f"
```

Reply:
I'm not sure if this will work because I used the windows `signtool.exe` and it looks like you are using a custom `.sh` script for the signing, so I can't guarantee this will work, since I did not test this configuration.